src/issueFinder.ts

```diff
@@ -25,17 +25,22 @@ export class IssueFinder {
         // TODO: loop through all the labels
         const label = labels.shift() || '';
         const result = await this.fetcher.fetchMatchingIssues(token, label, repositoryIdentifiers);
-        const issues = result.edges.map((edge) => {
-            const issue: Issue = {
-                title: edge.node.title,
-                createdAt: edge.node.createdAt,
-                repositoryNameWithOwner: edge.node.repository.nameWithOwner,
-                url: edge.node.url,
-            };
 
+        const issues = result.edges.map((edge) => {
+            const issue = this.convertFromGithubIssue(edge);
             return issue;
         });
 
         return issues;
     }
+    
+    private convertFromGithubIssue(edge: Edge): Issue {
```

Comment:
Since everything this function needs is inside the node, is there a reason to pass the edge instead of the node?

Reply:
just refactored the code and I'm passing the node